packages/cli/CHANGELOG.md

```diff
@@ -1,24 +1,31 @@
-# Change Log
-
-All notable changes to the Client code for "zowe-native-proto-cli" are documented in this file.
-
-Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
-
-## `0.1.1`
-
-- Fixed issue where a jobs list request returns unexpected results whenever a search query does not match any jobs. [#217](https://github.com/zowe/zowe-native-proto/pull/217)
-
-## `0.1.0`
-
-- Added `zssh restore dataset` command. [#38](https://github.com/zowe/zowe-native-proto/pull/38)
-- Added `zssh view uss-file` command. [#38](https://github.com/zowe/zowe-native-proto/pull/38)
-- Added `zssh list spool-files` command. [#38](https://github.com/zowe/zowe-native-proto/pull/38)
-- Added support for cancelling jobs. [#138](https://github.com/zowe/zowe-native-proto/pull/138)
-- Added support for holding and releasing jobs. [#182](https://github.com/zowe/zowe-native-proto/pull/182)
-- Updated error handling for listing data sets. [#185](https://github.com/zowe/zowe-native-proto/pull/185)
-- Added `zssh submit uss-file|local-file|data-set` commands. [#184](https://github.com/zowe/zowe-native-proto/pull/184)
-- Added support for conflict detection through use of e-tags. When a data set or USS file is viewed, the e-tag is displayed to the user and can be passed for future write requests to prevent overwriting new changes on the target system. [#144](https://github.com/zowe/zowe-native-proto/issues/144)
-
-## [Unreleased]
-
-- Initial release
+# Change Log
+
+All notable changes to the Client code for "zowe-native-proto-cli" are documented in this file.
+
+Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
+
+## Recent Changes
+
+- Added support for listing the long format for USS files with the `long` request option. When provided, the CLI plug-in prints additional file attributes, including permissions, file tag, file size, number of links, owner, and group. [#346](https://github.com/zowe/zowe-native-proto/issues/346)
+- Added support for listing all USS files with the `all` request option. When provided, the CLI plug-in includes hidden files in the list response. [#421](https://github.com/zowe/zowe-native-proto/pull/421)

```

Comment:
it seems that we get these by default 🤔 
<img width="289" height="178" alt="image" src="https://github.com/user-attachments/assets/20b7e33d-c1ff-4905-a181-a5eabf1331d8" />


even if you don't want them 😅 
<img width="357" height="176" alt="image" src="https://github.com/user-attachments/assets/5fde19d9-0ae3-41b7-8b6d-99723f0d89f9" />

Reply:
Let's discuss offline, I think you may have an old build of the server deployed 😅 

<img width="412" height="308" alt="image" src="https://github.com/user-attachments/assets/8fc143f3-3c70-4ef1-a87f-c1f10d18ddcf" />